src/lomc.jl

```diff
@@ -292,17 +292,21 @@ function lomc!(state::QMCState, df=DataFrame(); laststep=0, name="lomc!")
     # update_steps = 400
     @withprogress name=name while step < laststep
         step += 1
-        report!(state.r_strat, step, report, :steps, step)
+        if step % reporting_interval(state.r_strat) == 0
+            report!(state.r_strat, step, report, :steps, step)
+        end
         # This loop could be threaded if num_threads() == num_replicas? MPIData would
         # need to be aware of the replica's id and use that in communication.
         success = true
         for replica in state.replicas
             success &= advance!(report, state, replica)
         end
-        replica_names, replica_values = replica_stats(state.replica, state.replicas)
-        report!(state.r_strat, step, report, replica_names, replica_values)
-        report_after_step(state.r_strat, step, report, state)
-        ensure_correct_lengths(report)
+        if step % reporting_interval(state.r_strat) == 0
+            replica_names, replica_values = replica_stats(state.replica, state.replicas)
+            report!(state.r_strat, step, report, replica_names, replica_values)
+            report_after_step(state.r_strat, step, report, state)
+            ensure_correct_lengths(report)
+        end
```

Comment:
Just noting here that `report_after_step` is only called every `reporting_interval()`th time step. Please update the docstring!

Reply:
updated the docs for report_after_step